src/plugins/kibana_react/public/page/page_footer/page_footer.tsx

```diff
@@ -51,56 +51,65 @@ export const OverviewFooter: FC<Props> = ({ path }) => {
     defaultRoute === path ? (
       <RedirectAppLinks application={application}>
         <EuiButtonEmpty
-          iconType="home"
+          className="pageFooter__button"
```

Comment:
All the `pageFooter` classes need the `kbn` prefix and ideally would be more specific even though the name of the component is simply `PageFooter` because it's so general it could conflict in the future.